submissions/Articles/global-payouts-non-profits.md

```diff
@@ -0,0 +1,24 @@
+
+## How Global Payouts Can Make Your Transactions Quick
+
+As cliche as it is to say, “Time is money”, it truly is important for users to get their money on time. Sadly, traditional banking institutions tend to have long delivery processes. For example, [Wise users receives their payment depending on the country and currency](https://wise.com/help/articles/2978029/how-long-does-it-take-to-receive-money-into-a-balance). This can put nonprofits at a disadvantage as it can cause the communities they serve to financially help them in a timely manner.  Unlike their traditional counterparts, global payout systems don’t have a waiting period. Once person sends money, the other person gets its instantly. Nonprofits can benefit greatly from this as they don’t have to worry about the communities they serve not getting the funding they sent.  For instance, [AfricaHacks' collaboration with Chimoney for their hackathon rewards to the winners due to having a feature where they can send people money in just one click](https://chimoney.io/blogs/5-types-of-people-who-should-use-chimoneys-bulk-gift-cards-feature/).  Now before you go, there’s just one more benefit of using global payout systems for nonprofit funding.
+
+## How Global Payouts Can Give You Multiple Ways To Send Money
+
+Global payouts provide the necessary infrastructure for non-profits to send financial support quickly and efficiently. In contrast to traditional banking platforms, global payout platforms users have the freedom to use any method to send their money to other people. This can be helpful to nonprofits as it enables them to decide which delivery method best suits their organization and the people they serve. Case in point, [the International Student Identity Card (ISIC)'s partnership with Chimoney makes it easier for students manage their fiances](https://chimoney.io/blogs/chimoney-partners-with-isic-to-expand-global-benefits-for-chimoney-app-users/) as [they can send and receive money via email, their application, and text messaging](https://chimoney.io/payouts/).
+
```

Comment:
In this part could you add a section titled 'How Chimoney Empowers Communities and Non-profits'

In that section, please talk about the[ TryOpenGiving.com](https://tryopengiving.com/) platform which helps non-profits and communities around the world receive donations and get paid easily in various methods applicable to the donor through Chimoney. [Here's the repo of the platform for more info](https://github.com/Chimoney/non-profit-donations/tree/main/open-giving).

Kindly let me know if you need any help on this. Thanks